api/data_refinery_api/tests.py

```diff
@@ -203,15 +205,17 @@ def test_search_and_filter(self):
         self.assertEqual(response.json()['count'], 2)
 
         # Test search and filter
-        response = self.client.get(reverse('search'), {'search': 'THISWILLBEINASEARCHRESULT', 'technology': 'MICROARRAY'})
+        response = self.client.get(
+            reverse('search'), {'search': 'THISWILLBEINASEARCHRESULT', 'technology': 'MICROARRAY'})
         self.assertEqual(response.json()['count'], 1)
         self.assertEqual(response.json()['results'][0]['accession_code'], 'FINDME')
 
     def test_create_update_dataset(self):
 
         # Good
         jdata = json.dumps({'data': {"A": ["B"]}})
-        response = self.client.post(reverse('create_dataset'), jdata, content_type="application/json")
+        response = self.client.post(
+            reverse('create_dataset'), jdata, content_type="application/json")
```

Comment:
Did you get a new linter? I honestly don't think that this code looks improved this way, if you're going to line limit a function, then I think it's better to 
```python
response = self.client.post(
        endpoint=reverse('create_dataset'), 
        data=jdata, 
        content_type="application/json")
```
/opinion

Reply:
Kinda... mine was broken for a while without me realizing it. Once I realized and fixed it, it started correcting things all over the place. I agree that what you have there is more readable, I've been trying to catch places where it makes this correction and making it more like what you've got there.